crypto/bn/bn_exp.c

```diff
@@ -850,20 +850,21 @@ int BN_mod_exp_mont_consttime(BIGNUM *rr, const BIGNUM *a, const BIGNUM *p,
         top /= 2;
         bn_flip_t4(np, mont->N.d, top);
 
-        bits--;
-        for (wvalue = 0, i = bits % 5; i >= 0; i--, bits--)
-            wvalue = (wvalue << 1) + BN_is_bit_set(p, bits);
+        window0 = (bits - 1) % 5 + 1;
```

Comment:
Can you add a comment on this block of calculations?

Reply:
Harmonized, a.k.a. copy-pasted, even commentary.